silkrpc/common/constants.hpp

```diff
@@ -50,7 +50,7 @@ constexpr const char* kDefaultTarget{"localhost:9090"};
 constexpr const std::chrono::milliseconds kDefaultTimeout{10000};
 
 constexpr const char* kNodeName{"TurboGeth"};
-constexpr const char* kVersion{"2021.03.2-alpha"};
+constexpr const char* kVersion{"2021.05.3-alpha-3121264f"};
```

Comment:
The reason why Silkrpc response is different from Go RPCDaemon here is because Silkrpc is not within the Erigon repository so trying to follow the RPCDaemon Git commit seems not a good idea. TBH I already suggested to remove this info from RPCDaemon response as well, so please remove the commit value and use "2021.05.3-alpha"

Reply:
ok